crypto/keyring/keyring_test.go

```diff
@@ -29,69 +33,118 @@ func init() {
 	crypto.BcryptSecurityParameter = 1
 }
 
+func getCodec() codec.Codec {
```

Comment:
I declared `getCodec()` to fix cycle dep issue caused by importing `simapp`  that is used to call `simapp.MakeTestEncodingConfig().Codec`

Reply:
good, in fact we have a task to reduce `MakeTestEncodingConfig` usage.